app/code/Magento/Sales/etc/webapi.xml

```diff
@@ -10,271 +10,272 @@
     <route url="/V1/orders/:id" method="GET">
         <service class="Magento\Sales\Api\OrderRepositoryInterface" method="get"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::actions_view" />
         </resources>
     </route>
     <route url="/V1/orders" method="GET">
         <service class="Magento\Sales\Api\OrderRepositoryInterface" method="getList"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::actions_view" />
         </resources>
     </route>
     <route url="/V1/orders/:id/statuses" method="GET">
         <service class="Magento\Sales\Api\OrderManagementInterface" method="getStatus"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::actions_view" />
         </resources>
     </route>
     <route url="/V1/orders/:id/cancel" method="POST">
         <service class="Magento\Sales\Api\OrderManagementInterface" method="cancel"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::cancel" />
         </resources>
     </route>
     <route url="/V1/orders/:id/emails" method="POST">
         <service class="Magento\Sales\Api\OrderManagementInterface" method="notify"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::emails" />
         </resources>
     </route>
     <route url="/V1/orders/:id/hold" method="POST">
         <service class="Magento\Sales\Api\OrderManagementInterface" method="hold"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::hold" />
         </resources>
     </route>
     <route url="/V1/orders/:id/unhold" method="POST">
         <service class="Magento\Sales\Api\OrderManagementInterface" method="unHold"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::unhold" />
         </resources>
     </route>
     <route url="/V1/orders/:id/comments" method="POST">
         <service class="Magento\Sales\Api\OrderManagementInterface" method="addComment"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::comment" />
         </resources>
     </route>
     <route url="/V1/orders/:id/comments" method="GET">
         <service class="Magento\Sales\Api\OrderManagementInterface" method="getCommentsList"/>
         <resources>
-            <resource ref="Magento_Sales::sales" />
+            <resource ref="Magento_Sales::comment" />
```

Comment:
Should we use `Magento_Sales::actions_view` here, considering that `Magento_Sales::comment` is used only for adding comments currently?